lib/spack/spack/test/spec_semantics.py

```diff
@@ -328,6 +335,7 @@ def test_satisfies_single_valued_variant(self):
         # Assert that an autospec generated from a literal
         # gives the right result for a single valued variant
         assert 'foobar=bar' in a
+        assert 'foobar==bar' in a
```

Comment:
Does this just mean that `foobar=bar` satisfies `foobar==bar`? This seems like an odd place to be testing that, but if that's all it's testing it might be ok here.